### PR TITLE
DEVDOCS-2528: Embedded Checkout Domain Setup

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-](https://jira.bigcommerce.com/browse/DEVDOCS-2528)

## What changed?
* Added FAQ about domain setup requirement for safari compatibility.

### CODE REVIEW

**Comment by uwikaiddi** [comment]
@aglensmith, request a peer review for this

**Review by uwikaiddi — CHANGES_REQUESTED** [review]
@aglensmith, request a peer review for this

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 1]
```diff
@@ -7,33 +7,32 @@
...
 - [Logged-In customers](#logged-in-customers)
+- [FAQ](#faq)
 
 </div>
 
-Embedded Checkout lets you place BigCommerce’s checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
-
-<a id="how-it-works"></a>
+Embedded Checkout lets you place BigCommerce's checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:14]:
```suggestion
Embedded Checkout lets you place BigCommerce's checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for WordPress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 2]
```diff
@@ -7,33 +7,32 @@
...
 
 Embedded Checkout uses an HTML `<iframe>` to display BigCommerce's PCI compliant Optimized One-Page Checkout on non-BigCommerce web pages.
 
 If your channel site doesn't match the URL from which you're making a request to a BigCommerce store, you will get a security error and the checkout will not load. Additionally, if requests to your BigCommerce store aren't served over HTTPS, you will also see an error.
 
-<a id="channels-sites-and-routes-apis"></a>
-
 ## Channels, Sites, and Routes APIs
 
 You will need to use the [Channels, Sites, and Routes](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api) APIs to embed checkout on an external site. The Channels API allows you to create and manage sales channel listings across a merchant's product catalog. A channel can be a marketplace, like Amazon, or an external storefront, like a WordPress site. The Sites and Routes APIs let you set an external storefront domain and define the paths for important pages, like the home page, cart page, or checkout page. The Sites and Routes APIs allow you to link back to the proper URL from invoice emails and storefront links.
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:24]:
```suggestion
You will need to use the [Channels API](https://developer.bigcommerce.com/api-reference/store-management/channels) and the [Sites and Routes API](https://developer.bigcommerce.com/api-reference/store-management/sites) to embed checkout on an external site. The Channels API allows you to create and manage sales channel listings across a merchant's product catalog. A channel can be a marketplace, like Amazon, or an external storefront, like a WordPress site. The Sites and Routes API lets you set an external storefront domain and define the paths for important pages, like the home page, cart page, or checkout page. The Sites and Routes API allows you to link back to the proper URL from invoice emails and storefront links.
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 3]
```diff
@@ -7,33 +7,32 @@
...
 
-<a id="bigcommerce-checkout-sdk"></a>
-
 ## BigCommerce Checkout SDK
 
 Embedded Checkout requires the BigCommerce Checkout SDK to invoke a method to render the checkout in your site. Learn more about the [Checkout SDK](https://developer.bigcommerce.com/api-docs/cart-and-checkout/checkout-sdk).
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:28]:
```suggestion
Embedded Checkout requires the BigCommerce [Checkout SDK](https://developer.bigcommerce.com/api-docs/cart-and-checkout/checkout-sdk) to invoke a method to render the checkout in your site.
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 4]
```diff
@@ -7,33 +7,32 @@
...
 
-<a id="embedded-logged-in-customers"></a>
-
 ## Logged-In customers
 
 The [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api) allows you to manage customers in two steps. First, you need to pass the `customer_id` when creating the cart. Second, you need to log in the customer, so the session is active when the checkout loads.
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:32]:
```suggestion
The [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api) allows you to manage customers in two steps. First, you need to pass the `customer_id` when creating the cart. Second, you need to log the customer in, so the session is active when the checkout loads.
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 5]
```diff
@@ -7,33 +7,32 @@
...
+
+## FAQ
+
+### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:36]:
```suggestion
### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Error?**
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 6]
```diff
@@ -7,33 +7,32 @@
...
+
+In order for Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatibile with Safari.
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:38]:
```suggestion
For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center. 

```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md (hunk 1 above)
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:14]: Embedded Checkout lets you place BigCommerce's checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce control panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 7]
```diff
@@ -7,33 +7,32 @@
...
 
-<a id="bigcommerce-checkout-sdk"></a>
-
 ## BigCommerce Checkout SDK
 
 Embedded Checkout requires the BigCommerce Checkout SDK to invoke a method to render the checkout in your site. Learn more about the [Checkout SDK](https://developer.bigcommerce.com/api-docs/cart-and-checkout/checkout-sdk).
 
-<a id="embedded-logged-in-customers"></a>
-
 ## Logged-In customers
```
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:30]: ## Logged-in customers

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md [hunk 8]
```diff
@@ -116,7 +116,7 @@ This returns `id` which you will use as the `site_id` in future requests. The `u
 
 ## Creating a cart
 
-To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api)  `/cart` endpoint. 
+To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api)  `/cart` endpoint.
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:119]:
```suggestion
To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/store-management/carts)  `/cart` endpoint.
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md [hunk 9]
```diff
@@ -175,18 +175,20 @@ Next, generate cart redirect URLs by sending a `POST` request to `/carts/{{cart_
 ```
 
 ### Redirecting a logged-in customer to embedded checkout
+
 For some use cases, you may want your customer to log in before they can begin the checkout process.
 
-Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer). 
+Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer).
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:181]:
```suggestion
Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/storefront/customer-login-api).
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md [hunk 10]
```diff
@@ -252,6 +254,10 @@ If your channel site doesn't match the URL from which you're making a request to
 ### Are hosted payment gateways supported with Embedded Checkout?
 At this time, you cannot embed a checkout using a hosted payment gateway. See [Available Payment Gateways](https://support.bigcommerce.com/s/article/Available-Payment-Gateways#all-available) to determine which type of gateway you're using.
 
+### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:257]:
```suggestion
### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Error?**
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md (hunk 6 above)
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:38]: For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), you must serve your checkout page from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, your BigCommerce store's domain should be `checkout.mystore.com`. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari.

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md [hunk 11]
```diff
@@ -252,6 +254,10 @@ If your channel site doesn't match the URL from which you're making a request to
...
+
+In order for Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatibile with Safari.
```
Comment (ektravel) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:259]:
```suggestion
For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center. 
```

**Review by ektravel — COMMENTED** [review]
Added my comments.

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md [hunk 12]
```diff
@@ -71,7 +71,7 @@ The response will contain an `id` which we will use as the `channel_id` in futur
 <div class="HubBlock--callout">
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
-    
+
 <!-- theme:  -->
 > Channels created via API are visible in the BigCommerce store's Control Panel in **Products** > **Listed On**. The Orders section will now also include a filter for your channel.
```
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:76]: > Channels created via API are visible in the BigCommerce store's control panel in **Products** > **Listed On**. The Orders section will now also include a filter for your channel.

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md (hunk 8 above)
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:119]: To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api) `/cart` endpoint.

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md (hunk 9 above)
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:181]: Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/storefront/customer-login-api).

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md (hunk 10 above)
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:257]: ### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" errors?

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md (hunk 11 above)
Comment (traci707) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:259]: For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), you must serve your checkout page from the same domain as your BigCommerce storefront. For example, if your headless storefront is www.mystore.com, your BigCommerce store's domain should be checkout.mystore.com. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari.

**Review by traci707 — CHANGES_REQUESTED** [review]

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 13]
```diff
@@ -7,33 +7,32 @@
...
 - [Logged-In customers](#logged-in-customers)
+- [FAQ](#faq)
 
 </div>
 
-Embedded Checkout lets you place BigCommerce’s checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
-
-<a id="how-it-works"></a>
+Embedded Checkout lets you place BigCommerce's checkout onto any website. Customers can check out on an external storefront while their order information syncs simultaneously to the BigCommerce Control Panel. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for WordPress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
```
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:14]: @aglensmith, the statements made here and the reference to BC4WP may no longer be true.
See https://bigcommerce.slack.com/archives/C57SQ401K/p1614112903001300.

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 14]
```diff
@@ -7,33 +7,32 @@
...
-
-<a id="bigcommerce-checkout-sdk"></a>
+You will need to use the [Channels API](https://developer.bigcommerce.com/api-reference/store-management/channels) and the [Sites and Routes API](https://developer.bigcommerce.com/api-reference/store-management/sites) to embed checkout on an external site. The Channels API allows you to create and manage sales channel listings across a merchant's product catalog. A channel can be a marketplace, like Amazon, or an external storefront, like a WordPress site. The Sites and Routes API lets you set an external storefront domain and define the paths for important pages, like the home page, cart page, or checkout page. The Sites and Routes API allows you to link back to the proper URL from invoice emails and storefront links.
```
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:24]:
```suggestion
You will need to use the [Channels API](https://developer.bigcommerce.com/api-reference/store-management/channels) and the [Sites and Routes API](https://developer.bigcommerce.com/api-reference/store-management/sites) to embed a checkout on an external site. The Channels API allows you to create and manage sales channel listings across a merchant's product catalog. A channel can be a marketplace, like Amazon, or an external storefront, like a WordPress site. The Sites and Routes API lets you set an external storefront domain and define the paths for important pages, like the home page, cart page, or checkout page. The Sites and Routes API allows you to link back to the proper URL from invoice emails and storefront links.
```

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

**Comment by uwikaiddi** [comment]
@aglensmith, still waiting on these updates.

**Review by markcmurphy — DISMISSED** [review]

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 15]
```diff
@@ -7,33 +7,32 @@
...
 
-<a id="embedded-logged-in-customers"></a>
-
 ## Logged-In customers
 
 The [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api) allows you to manage customers in two steps. First, you need to pass the `customer_id` when creating the cart. Second, you need to log in the customer, so the session is active when the checkout loads.
+
+## FAQ
+
+### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**
```
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:36]:
```suggestion
### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 16]
```diff
@@ -7,33 +7,32 @@
...
-
-<a id="how-it-works"></a>
+The [Checkout SDK's](https://github.com/bigcommerce/checkout-sdk-js) [`embedded-checkout` sub-module](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/README.md#embedcheckout) can be used to embed [BigCommerce's Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout) into non-native storefronts like WordPress. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. View more information about the plugin in the article [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
```
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:14]:
```suggestion
The [Checkout SDK's`embedded-checkout` sub-module](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/README.md#embedcheckout) can be used to embed [BigCommerce's Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout) into non-native storefronts like WordPress. You can see this in action within the BigCommerce for WordPress plugin, which uses the same process described here as a checkout option for merchants. For more information about the plugin, see [BigCommerce for Wordpress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/getting-started/introduction).
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-overview.md [hunk 17]
```diff
@@ -7,33 +7,32 @@
...
-
 ## Logged-In customers
 
 The [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api) allows you to manage customers in two steps. First, you need to pass the `customer_id` when creating the cart. Second, you need to log in the customer, so the session is active when the checkout loads.
+
+## FAQ
+
+### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**
+
+For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center.
```
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-overview.md:38]:
```suggestion
For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari).
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md (hunk 10 above)
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:257]:
```suggestion
### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?
```

File: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md [hunk 18]
```diff
@@ -252,6 +254,10 @@ If your channel site doesn't match the URL from which you're making a request to
 ### Are hosted payment gateways supported with Embedded Checkout?
 At this time, you cannot embed a checkout using a hosted payment gateway. See [Available Payment Gateways](https://support.bigcommerce.com/s/article/Available-Payment-Gateways#all-available) to determine which type of gateway you're using.
 
+### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**
+
+For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center.
```
Comment (uwikaiddi) [inline docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md:259]:
```suggestion
For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari).
```

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**